exporter/fileexporter/file_exporter_test.go

```diff
@@ -132,3 +127,17 @@ func tempFileName(t *testing.T) string {
 	require.NoError(t, os.Remove(socket))
 	return socket
 }
+
+// ErrorWriter is an io.Writer that will return an error all ways
+type ErrorWriter struct {
+}
+
+func (e ErrorWriter) Write(p []byte) (n int, err error) {
+	return 0, errors.New("all ways return error")
+}
+
+func (e *ErrorWriter) Close() error {
+	return nil
+}
+
+var _ io.WriteCloser = new(ErrorWriter)
```

Comment:
Can be removed, already used as WriteCloser in the exporter so it is guarantee that implements the interface.